frontend/src/components/Header.tsx

```diff
@@ -1,79 +1,113 @@
+          mobileMenuOpen ? 'translate-x-0' : '-translate-x-full'
+        )}
+      >
+        <div className="space-y-1 px-2 pb-3 pt-2">
+          {/* Logo */}
           <NavLink to="/">
             <div className="flex h-full items-center">
               <img
-                src={!dark ? '/img/owasp_icon_black_sm.png' : '/img/owasp_icon_white_sm.png'}
-                className="h-16 px-2"
+                src={'/img/owasp_icon_white_sm.png'}
+                className="hidden h-16 dark:block"
+                alt="OWASP Logo"
+              ></img>
+              <img
+                src={'/img/owasp_icon_black_sm.png'}
+                className="block h-16 dark:hidden"
                 alt="OWASP Logo"
               ></img>
               <div className="text-2xl text-slate-800 dark:text-slate-300 dark:hover:text-slate-200">
                 Nest
               </div>
             </div>
           </NavLink>
-          <div className="flex-1 justify-between rounded-lg pl-6 font-medium">
-            <div className="flex justify-start pl-6 text-slate-700 dark:text-slate-300">
-              {headerLinks.map((link, i) => (
-                <NavLink
-                  key={i}
-                  to={link.href}
-                  className="navlink px-3 py-2 hover:text-slate-800 dark:hover:text-slate-200"
-                  aria-current="page"
-                >
-                  {link.text}
-                </NavLink>
-              ))}
-            </div>
-          </div>
-          <div className="justify-end">
-            <label className="inline-flex cursor-pointer content-center items-center">
-              <span className="ms-3 pr-2 text-sm font-medium text-gray-900 dark:text-gray-300">
-                {icon}
-              </span>
-              <input
-                onChange={toggleTheme}
-                type="checkbox"
-                value=""
-                className="peer sr-only"
-              ></input>
-              <div className="peer relative h-5 w-9 rounded-full bg-gray-200 after:absolute after:start-[2px] after:top-[2px] after:h-4 after:w-4 after:rounded-full after:border after:border-gray-300 after:bg-white after:transition-all after:content-[''] peer-checked:bg-slate-500 peer-checked:after:translate-x-full peer-checked:after:border-white rtl:peer-checked:after:-translate-x-full dark:border-gray-500 dark:bg-gray-700"></div>
-              <span className="ms-3 text-sm font-medium text-gray-900 dark:text-gray-300">
-                <i className="fa-regular fa-moon"></i>
-              </span>
-            </label>
-          </div>
+          {headerLinks.map((link, i) => (
+            <NavLink
+              key={i}
+              to={link.href}
+              className={cn(
+                'block px-3 py-2 text-slate-700 hover:text-slate-800 dark:text-slate-300 dark:hover:text-slate-200',
```

Comment:
Also, can we put the `navlink` class back? it is a custom class that addresses link color for navigation. Right now active link looks

Reply:
I have added the navlink now